### PR TITLE
[Alerts] Convert event value to be dict to allow multiple values

### PR DESCRIPTION
**Pull request by yanburman**
Add more values to the model monitoring event that would be of interest for the user

https://iguazio.atlassian.net/browse/ML-6233

### CODE REVIEW

File: mlrun/model_monitoring/writer.py [hunk 1]
```diff
@@ -227,10 +232,20 @@ def do(self, event: _RawEvent) -> None:
         _Notifier(event=event, notification_pusher=self._custom_notifier).notify()
 
         if mlrun.mlconf.alerts.mode == mlrun.common.schemas.alert.AlertsModes.enabled:
+            endpoint_record = get_endpoint_record(
```
Comment (davesh0812) [inline mlrun/model_monitoring/writer.py]: I think it is nice to have the model name in the message, but I think we don't want to access the db in each writer run. Try to save all the model_endpoint you already get from the db like we are doing in the `EventStreamProcessor::ProcessEndpointEvent.resume_state`.

**Review by davesh0812 — COMMENTED** [review]
Looks good!
I had one comment on the implementation. 
About the testing maybe we can had in the system test a check to ensure the message is has we expected, if it is possible.

**Review by davesh0812 — APPROVED** [review]
LGTM

**Review by theSaarco — APPROVED** [review]
LGTM.

**Review by liranbg — APPROVED** [review]

**Pull request merged**